src/EnergyPlus/ScheduleManager.cc

```diff
@@ -295,6 +297,11 @@ namespace Sched {
         auto *weekSched = new WeekSchedule;
         weekSched->Name = name;
 
+        // Fill the dayScheds with the Missing Day Schedule (Always Off)
+        for (int iDayType = 1; iDayType < (int)DayType::Num; ++iDayType) {
+            weekSched->dayScheds[iDayType] = s_sched->daySchedules[SchedNum_AlwaysOff];
+        }
```

Comment:
AddWeekSchedule initializes the dayScheds to that missing schedule (except the "Unused" day type...)